src/Builder.jl

```diff
@@ -76,7 +76,7 @@ Selectors.matcher(::Type{T}, doc::Documents.Document) where {T <: DocumentPipeli
 Selectors.strict(::Type{T}) where {T <: DocumentPipeline} = false
 
 function Selectors.runner(::Type{SetupBuildDirectory}, doc::Documents.Document)
-    Utilities.log(doc, "setting up build directory.")
+    @info "Documenter: setting up build directory."
```

Comment:
Hmm, now these lines are even more noisy.

```
[ Info: Documenter: setting up build directory.
[ Info: Documenter: expanding markdown templates.
[ Info: Documenter: building cross-references.
```

Reply:
Yea, I'm dropping the prefix.